AASwiftSDK/Public/AAZoneView.swift

```diff
@@ -242,12 +251,27 @@ import WebKit
         }
         currentAdView = newAdView
 
-        if let currentAdView = currentAdView {
-            addSubview(currentAdView)
-        }
         AASDK.logDebugFrame(self.frame, message: "AAZoneView \(zoneId ?? "") rendering ad. The AAZoneView's frame is")
         let frame = CGRect(x: 0, y: 0, width: self.frame.size.width, height: self.frame.size.height)
         currentAdView?.frame = frame
+
+        reportAdView.setImage(UIImage(named: "reportAdIcon", in: Bundle(for: AAZoneView.self), compatibleWith: nil), for: .normal)
+
+        if let adId = ad?.adID, let uid = UserDefaults.standard.value(forKey: AA_KEY_UDID) {
+            reportAdUrl = "https://feedback.add-it.io/?aid=\(adId)$uid=\(uid)"
```

Comment:
It doesn't look like the url is encoded properly? When viewing it in the browser I see literal :: marks when I would expect something more like this: `https://dev.feedback.add-it.io/?aid=1407%3A%3A102110%3A%3A300%3A%3A1287%3A%3A6345%3A%3A2814&uid=b0bee5c4-3d17-4f60-90b8-36e4e05e981c`. Also I think `$` should be `&` instead and I forgot to mention we need a dev variant as well to go with the prod: https://dev.feedback.add-it.io/

Also for safety sake can we put this in a const with the other URLs and parameterize this with a builder or something?